proxy.php

```diff
@@ -89,9 +89,6 @@ public function checkRequest()
 		if (md5($request . $this->secret) != $hash)
 			return false;
 
-		// Ensure any non-ASCII characters in the URL are encoded correctly
-		$request = iri_to_url($request);
```

Comment:
Don't remove this line. Instead, all you should need to do is change it to:
```
$request = iri_to_url(rawurldecode($request));
```

Reply:
Disregard my previous comment. The issue was just that iri_to_url() wasn't correctly handling double-encoded `%` characters. #5194 fixes it properly.